05/ukol6_version2.py

```diff
@@ -0,0 +1,20 @@
+#Had byl pyšný na to, že je v abecedě první. Dokud nepřiletěla "andulka".
+#Abys hada uklidnila, vytvoř program, který zvířata seřadí podle abecedy, ale bude
+#ignorovat první písmeno
+#(t.j. vypíše ["had", "pes", "andulka", "kočka", "králík"])
+
+list = ['pes', 'kocka', 'kralik', 'had',]
+list.append('andulka')
+dictionary = {}
+result = []
+
+
+for word in list:
+    dictionary[word[1:]] = word                 #V dictionary vytvari slovnik (slovo bez prvniho pismene: puvodni slovo)
+sorted_dict = dict(sorted(dictionary.items()))  #Vytvori novy slovnik serazeny podle abecedy.
+
+for value in sorted_dict.values():              #Kazda hodnota ze sorted_dict se ulozi do seznamu result.
+    result.append(value)
+
+print(result)
```

Comment:
Mno, hezky 😄 A jako bonus se podívej, jak by to třeba šlo strašně moc zjednodušit, kdybyste už věděly o něco víc (např. o [lambda funkcích](https://docs.python.org/3/reference/expressions.html#lambda)): https://github.com/Milis85/pyladies/pull/2#discussion_r1004112107 😉